Panic: nil font in paragraph measureWords with nested headings

## Description

Rendering certain HTML templates with `<h3>` headings nested inside styled divs causes a nil pointer dereference in `font.(*Standard).MeasureString`. The panic occurs because a `TextRun` is created with both `Font` and `Embedded` set to nil.

## Stack trace

```
panic: runtime error: invalid memory address or nil pointer dereference
goroutine 1 [running]:
github.com/carlos7ags/folio/font.(*Standard).MeasureString(0x0, ...)
  font/metrics.go:73
github.com/carlos7ags/folio/layout.(*Paragraph).measureWords(...)
  layout/paragraph.go:1003
github.com/carlos7ags/folio/layout.(*Heading).PlanLayout(...)
  layout/heading.go:170
```

## Reproduction

The marketing flyer template in the playground triggers this. Key HTML pattern:

```html
<div class="feature-card">
  <h3>HTML to PDF</h3>
  <p>Description text</p>
</div>
```

## Fix

`runMeasurer()` in `layout/paragraph.go` returns `run.Font` without nil check. When a `TextRun` has no font assigned (e.g., from certain converter code paths), this returns nil which panics on the next `MeasureString` call.

Defensive fix: fall back to `font.Helvetica` when both `Embedded` and `Font` are nil.

## Impact

- WASM playground crashes with "Go program has already exited"
- Any HTML with headings in nested containers can trigger this
- Introduced in v0.6.0 (baseline positioning changes)

--- a/layout/paragraph.go
+++ b/layout/paragraph.go
@@ -402,7 +402,11 @@ func runMeasurer(run TextRun) font.TextMeasurer {
 	if run.Embedded != nil {
 		return run.Embedded
 	}
-	return run.Font
+	if run.Font != nil {
+		return run.Font
+	}
+	// Fallback: use Helvetica if no font is set (defensive).
+	return font.Helvetica
 }
 
 // computeBaseline returns the distance from the top of the line box to the

